### DISCUSSION TITLE
."This user account is blocked. Contact registration@vercel.com for more information."

### DISCUSSION THREAD

**Discussion by DOT6474** [vercel/community#1280] [Help] [answered]
### Summary

![image](https://user-images.githubusercontent.com/122854605/212817282-36155cec-4a92-47ec-9f8b-6b5abe3bf784.png)
Is there a way to solve it?

**Comment by swarnava** [comment, marked as answer]
Yes, you can contact to registrations@vercel.com as error suggested. :)
